CHANGELOG.mdx

```diff
@@ -17,6 +17,16 @@ https://github.com/RasaHQ/rasa/tree/main/changelog/ . -->
 
 <!-- TOWNCRIER -->
 
+## [3.10.0.dev1] - 2024-08-20
```

Comment:
@sanchariGr  Should it be like this?

Reply:
We dont release changelog for pre releases, I think maybe the condition in the script did not account for dev releases.

Reply:
I will revert this then.